python/ray/serve/schema.py

```diff
@@ -1103,6 +1115,11 @@ class ServeInstanceDetails(BaseModel, extra=Extra.forbid):
     )
     target_capacity: Optional[float] = TARGET_CAPACITY_FIELD
 
+    target_details: Dict[str, List[TargetInfo]] = Field(
```

Comment:
following the existing convention
```suggestion
    targets: Dict[str, List[TargetDetails]] = Field(
```